packages/neon/neon/lib/src/app.dart

```diff
@@ -266,7 +266,7 @@ class _NeonAppState extends State<NeonApp> with WidgetsBindingObserver, tray.Tra
   }
 
   @override
-  Widget build(final BuildContext context) => StreamBuilder<Brightness>(
+  Widget build(final BuildContext context) => StreamBuilder<Brightness?>(
```

Comment:
This breaks the app that the NativeSplash never gets removed.
the blow `!platformBrightnessSnapshot.hasData` internally checks wether `platformBrightnessSnapshot.data != null` thus we have a problem here

Reply:
I already stumbled over this another time and IMO we should just generate a `darkTheme` and a `lightTheme` and let MaterialApp decide the brightness for us.

---

A bit unrelated I once played with making `nextcloudTheme` a ThemeExtension as this would enable us to use the usual Theme.of(context).extesion<CustomExtenstion>(); to get our theme everywhere we have our beloved BuidContext. Sadly this just added more boilerplate than was worth for me to really submit.

Reply:
Sounds good. I forgot to check this change, I knew that it could break things.

Reply:
I could make a change that we land before this PR as the theme changes isn't related to the update.

Reply:
Sure, I'd appreciate that. We would have to merge it with failing CI though :)